packages/server/graphql/mutations/removeAgendaItem.ts

```diff
@@ -34,7 +34,10 @@ export default {
     const agendaItem = await r
       .table('AgendaItem')
       .get(agendaItemId)
-      .delete({returnChanges: true})('changes')(0)('old_val')
```

Comment:
no more hard deletes on agenda items, this was causing dataloaders to fail the lookup